DESCRIPTION

```diff
@@ -23,10 +23,15 @@ Description: Provides functionality for installing and updating
     github only packages. Other functionalities like checking conflicts
     are also provided. Inspired by tidyverse and dtupdate packages.
 License: GPL-3
-URL: https://github.com/natverse/natverse
+Encoding: UTF-8
+LazyData: true
+RoxygenNote: 7.0.2
+Language: en-GB
+URL: https://github.com/natverse/natverse, https://jefferis.github.io/nat/
```

Comment:
Can you change the url  https://jefferis.github.io/nat/ to https://natverse.github.io/nat/  as the previous one doesn't exist.

Reply:
Okay!